actionpack/lib/action_dispatch/middleware/exception_wrapper.rb

```diff
@@ -174,6 +174,31 @@ def self.status_code_for_exception(class_name)
       Rack::Utils.status_code(@@rescue_responses[class_name])
     end
 
+    def show?(request)
+      # We're treating `nil` as "unset", and we want the default setting to be
+      # `:all`. This logic should be extracted to `env_config` and calculated
+      # once.
+      config = request.get_header("action_dispatch.show_exceptions")
+
+      # For backwards compatibility.
+      if config == true
```

Comment:
This might be an unpopular opinion but when I see multiple if/elsif checking the value of the same thing, I feel like a switch/cond is better 🤔